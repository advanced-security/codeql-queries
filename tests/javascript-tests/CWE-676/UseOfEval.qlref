CWE-676/UseOfEval.ql