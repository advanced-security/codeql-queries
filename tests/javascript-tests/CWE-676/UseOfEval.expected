| app.js:2:6:2:12 | "1 + 1" | Using eval |
